pkg/nodes/node.go

```diff
@@ -23,10 +25,77 @@ const (
 
 // Builder provides struct for Node object containing connection to the cluster and the list of Node definitions.
 type Builder struct {
-	Definition *v1.Node
-	Object     *v1.Node
-	apiClient  *clients.Settings
-	errorMsg   string
+	Definition  *v1.Node
+	Object      *v1.Node
+	apiClient   *clients.Settings
+	errorMsg    string
+	drainHelper *drain.Helper
+}
+
+// NewDrainHelper builds drain Helper that contains parameters to control the behaviour of drain.
+func (builder *Builder) NewDrainHelper(
```

Comment:
Let's change a bit API.
I would recommend to Define following function:

SetDrainHelper which'll actually configure drainHelper *drain.Helper object based on given parameters.
setDefaultDrainHelper - private function that configures *drain.Helper based on some "best practices" configuration.

No need to add "node" for the following public functions because the object itself is a node.
Drain
Cordon
Uncordon

For the functions above please add simple condition.
if builder.drainHelper == nil{
    setDefaultDrainHelper()
}

Reply:
`NewDrainHelper` renamed to `SetDrainHelper`

WRT `setDefaultDrainHelper` I'm not sure where to plug it in as there's no a method that create _Node_ object, only the one that pulls it in or the one that lists nodes. Could U please elaborate on how to achieve it? Thanks!